src/test/java/org/scijava/script/ScriptInfoTest.java

```diff
@@ -285,6 +287,11 @@ public ScriptEngine getScriptEngine() {
 		public List<String> getExtensions() {
 			return Arrays.asList("bsizes");
 		}
+
+        @Override
+        public AutoCompleter getAutoCompleter(ScriptEngine scriptEngine) {
+            return new DefaultAutoCompleter(this, scriptEngine);
```

Comment:
This override is unnecessary, no? Because `AbstractScriptLanguage` already defines this same logic?